src/vaults/legacy/LegacyVaultAdapter.sol

```diff
@@ -36,7 +37,7 @@ contract LegacyVaultAdapter is AsyncVault, ILegacyVaultAdapter, IInvestmentManag
         address asset,
         IShareToken token,
         address root,
-        address manager,
+        IAsyncRedeemManager manager,
```

Comment:
CR: Should be `IInvestmentManager `

Reply:
Same answer as above, the vault does not use more capabilities than `IAsyncRedeemManager` offer

Reply:
🤔 After tried I realized I can not use `IBaseInvestmentManager` because the underlying vault is the `AsyncVault` who use `IAsyncRedeemManager` actions

Reply:
This is the legacy vault, so it must be `IInvestmentManager`

Reply:
Ahh, ok, using the old interface
Correct! I read you bad